manifests/h/Huawei/QuickAppIde/14.0.1/Huawei.QuickAppIde.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Huawei.QuickAppIde
+PackageVersion: 14.0.1
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- card
+- ux
+ReleaseDate: 2024-03-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://contentcenter-vali-drcn.dbankcdn.cn/pvt_2/DeveloperAlliance_package_901_9/5e/v3/aFdD1fSZSOOrRtqsWG14Pw/QuickAppIDE-V14.0.1-Win64.exe?HW-CC-KV=V1&HW-CC-Date=20240306T073107Z&HW-CC-Expire=315360000&HW-CC-Sign=0422592C5622FEA1EAE520EDEDBBBB2BBEB95053B96178BC1FB66481C85174E9
```

Comment:
```suggestion
  InstallerUrl: https://appfile-cn.dbankcdn.com/FileServer/getFile/appAttach/011/111/111/0000000000011111111.20221114162359.00120356616353505553714097760392:20471231000000:0001:0A825835C422CD1C421C0D46615896E8A6339C6BECCA31651023273FBC514B34.exe?needInitFileName=true
```

Reply:
The Chinese site. The English one is a bit outdated.